public/content/lessons/2017/derivative-formulas-geometrically/index.mdx

```diff
@@ -1,10 +1,381 @@
+
+<!-- TODO: link to binomial expansion -->
+
+Focusing on the rate in change introduced by the very small nudge, highlighted in red, and ignoring expressions containing $dx^2$, the pattern that emerges is what's known in the business as the "power rule". Given a monomial function raised to some power, $n$, applying the rule gives us the derivative of the function.
+
+<Figure
+  image="figures/7.32.svg" 
+  caption="Power Rule Definition"
+/>
+
+Even though in practice you will find yourself performing this derivative quickly and symbolically, imagining that exponent hopping down to the front, every now and then it’s nice to step back and remember why this rule works. Not just because it’s pretty, and not just because it helps to remind us that math actually makes sense and isn’t just a pile of formulas to memorize, but because it flexes that very important muscle of thinking about derivatives in terms of tiny nudges.
+
+<Question
+  question="What is the derivative of the function $f(x)= 3x^2$?"
+  choice1="$2x$"
+  choice2="$5x$"
+  choice3="$3x$"
+  choice4="$6x$"
+  answer={4}
+>
+
+This question can be modeled and solved as a question of change in area. For example, nudging the input of the function by $dx$ gives the expression $3(x+d x)^{2}$ which expands to the expression $3(x + dx)( x + dx)$. I've drawn the area below where the $3$ scales one of the sides.
+
+<Figure
+  image="figures/power-rule-example.svg" 
+  caption="Derivative of $f(x) = 3x^{2}$ modeled as area"
+/>
+
+Looking at the figure we can see the rate of the change of the function represented by the new area of $6$ thin rectangles with area $x \cdot dx$. Alternatively, the question can be modeled as volume, where instead of scaling one of the sides, the $3$ extends the geometry into another dimension.
+
+<Figure
+  image="figures/3x^2-volume-model.svg" 
+  caption="Derivative of $f(x) = 3x^{2}$ modeled as volume"
+/>
+
+This gives us the equivalent derivative, as the new volume of the two rectangular prisms is equal to $3 \cdot x \cdot dx + 3 \cdot x \cdot dx = 6x$. As before, we are ignoring the smaller rectangular prism because its volume $3dx^2$ contains $dx^2$.
+
+</Question>
+
+### Derivative of $f(x) = \frac{1}{x}$
+
+As another example, think of the function $f(x) = \frac{1}{x}$. Now, on the one hand, you could blindly try applying the power rule, since $\frac{1}{x}$ is the same as writing $x^{-1}$. That would involve letting that $-1$ hop down, leaving behind one less than itself, $-2$. 
+But let’s have some fun and see if we can reasonable this geometrically, rather than just plugging it through a formula.
+
+The value $1/x$ is asking “what number multiplied by $x$ equals $1$”, so here’s how I’d visualize it: Imagine a little rectangular puddle of water in two dimensions with area $1$. Let’s say that it’s width is $x$, which means the other must be $1/x$, since the total area is $1$.
```

Comment:
```suggestion
The value $1/x$ is asking “what number multiplied by $x$ equals $1$”, so here’s how I’d visualize it: Imagine a little rectangular puddle of water in two dimensions with area $1$. Let’s say that it’s width is $x$, which means its height must be $1/x$, since the total area is $1$.
```